### ISSUE TITLE
[Components and pattern standards] Navigation mechanisms aren't consistent across pages. (04.10.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#55856]
#### VFS team name
VA Online Scheduling (VAOS)

#### VFS product name
VA Online Scheduling

#### VFS feature name
Appointment List

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design
Brian DeConinck - @briandeconinck - Accessibility
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Navigation mechanisms aren't consistent across pages.

### Issue Details
Hanging navigation links appear associated with the appointments list, however when a user goes to pending they disappear. The user does not know what page they are on or how to get back to the previous page. Also, the hanging links to not currently indicate to the user which page they are already on.

Additionally, the hanging navigation is coded as a `nav` landmark. Screen reader users often navigate the page by exploring landmark regions, and would typically expect the landmarks to stay consistent within a section of related pages on VA.gov. Having the `nav` present on one page but missing on other pages may lead to some confusion for screen reader users.

### VA.gov Experience Standard
[Category Number 04, Issue Number 10](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.3_AA

### Platform Recommendation
Design a navigation component that can do what you need. Work with the experimental design process.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by outerpress** [comment]
Just to clarify the nuance between these two tickets for my own notes:
- This ticket adds the subnavigation to the past and pending lists
- Ticket #55862 adds the page the user is currently on to the subnavigation

**Comment by sterkenburgsara** [comment]
@outerpress Totally fair question! Both of these tickets describe the same issue, but ask you to do different things. Admittedly, we made this confusing in our tickets and I apologize for that. Here's what we intended:
* This ticket asks you to design a navigational element (through the experimental design process) that has three navigational links (upcoming, pending, past), and makes the current state clear to end-users. 
* The other ticket #55862 asks you to add that navigational element (once designed) to all pages within the experience, reflecting the current state (which will change / be dependent on which page the user is on). We'd also like you to encode it as attached to the appointments lists, so that it doesn't get visually separated from the listed appointments if any alerts appear on the page.

**Comment by outerpress** [comment]
Thank you @sterkenburgsara !

**Comment by ldelacosta** [comment]
@shiragoodman - We completed this work in [#56167](https://app.zenhub.com/workspaces/vaos-team-603fdef281af6500110a1691/issues/gh/department-of-veterans-affairs/va.gov-team/56167). cc: @JRRoof
